src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Modules/MetadataEndpointModule.cs

```diff
@@ -45,7 +45,7 @@ internal async Task GetMetadata(HttpContext httpContext)
         {
             dependencies = $"{baseUrl}/dependencies",
             specifications = $"{baseUrl}/specifications",
-            xsdFiles = $"{baseUrl}/xsdMetadata"
+            xsdMetadata = $"{baseUrl}/xsd"
```

Comment:
This is the format I had previously defined in the Discovery API standard. I was able to find this error thanks to @elopezgap's review of the API standard documents.